Plugins/NativeInput/Source/NativeInput.cpp

```diff
@@ -11,7 +11,7 @@ namespace Babylon::Plugins
     {
         constexpr auto JS_NATIVE_INPUT_NAME = "_nativeInput";
 
-        constexpr auto POINTER_BASE_DEVICE_ID = "Pointer";
+        //constexpr auto POINTER_BASE_DEVICE_ID = "Pointer";
```

Comment:
This looks like it was left over from the earlier contract that used string ids? Remove?

Reply:
I had to comment this out as it was causing a warning during BabylonReactNative compile (oddly not during regular BabylonNative compile), and since we have warnings as errors turned on this was breaking the build.  I didn't remove it because I was not sure if it was planned to be used in the future, but with your vote I'll go ahead and take it out.